rasa_core/train.py

```diff
@@ -232,6 +233,9 @@ def train_dialogue_model(domain_file, stories_file, output_path,
     _interpreter = NaturalLanguageInterpreter.create(cmdline_args.nlu,
                                                      _endpoints.nlu)
 
+    domain = TemplateDomain.load(cmdline_args.domain)
+    _tracker_store = TrackerStore.find_tracker_store(domain,
```

Comment:
same here, better to set the `domain` to `None` and not load it

Reply:
I removed the domain dependency and changed it to None, ran the tests again and it was fine.

Reply:
thanks a lot :+1: